packages/scss/src/components/_button.scss

```diff
@@ -28,11 +29,29 @@
 	}
 }
 
-@mixin buttonLinkColoring($palette) {
+@mixin buttonLinkColoring($palette, $activeShadowPalette) {
+	color: _color($palette, 700);
+
 	&:not([disabled], .is-disabled) {
+		&:hover {
+			background-color: _color($palette, 50);
+			box-shadow: none;
+		}
+
+		&:focus,
+		&:active {
```

Comment:
Can we have it inside the focus and the active declaration instead of having focus and active declared two times? (Easier for readers)